### PR TITLE
re-order filter for non identified  indiv

### PR DESCRIPTION
**Pull request by ktalbi**
115927475

### CODE REVIEW

**Comment by romfabbro** [comment]
dot comma required at line 13

File: Back/database/Pipe/DB_Mother/075_Up_indiv_non_identif_reorganize_filter.txt [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+WHERE Module_ID = 8 and Name = 'Sex'
+
+
+Insert into ModuleGrids
+	(Module_ID, TypeObj, Name, Label, GridRender, GridSize, CellType, GridOrder, QueryName, Options, FilterOrder, FilterSize, FilterClass, IsSearchable, 
+                         FilterDefaultValue, FilterRender, FilterType, Status)
+
+SELECT Module_ID, TypeObj, Name, Label, GridRender, GridSize, CellType, GridOrder, QueryName, Options, FilterOrder, FilterSize, FilterClass, IsSearchable, 
+                         FilterDefaultValue, FilterRender, FilterType, Status
+FROM ModuleGrids WHERE Module_ID = 8 and Name = 'Sex'
```
Comment (romfabbro) [inline Back/database/Pipe/DB_Mother/075_Up_indiv_non_identif_reorganize_filter.txt]: put " ; " at the end of this line

**Comment by ktalbi** [comment]
done

**Comment by romfabbro** [comment]
ok

**Pull request merged**